skipchain/skipchain.go

```diff
@@ -1962,3 +1969,16 @@ func (s *Service) incrementWorking() error {
 
 	return nil
 }
+
+// SetSkipChainValidPeers is a utility function to assign the set of valid
+// peers for the given skipchain.
+func SetSkipChainValidPeers(ctx *onet.Context, scID SkipBlockID,
+	validPeers []*network.ServerIdentity) {
+	// Compute the PeerSetID as hash(serviceID | skipchainID)
```

Comment:
Sorry - missed that in your last PR. My proposition was to have this hash done in `NewPeerSetID`, so that the services don't have to create their own domain-protection, but can rely on `Context` doing the right thing.

Reply:
But the SkipChain ID, being not known by `onet`, must still be provided, isn't it?

Reply:
yes, but not the serviceID.